app/code/Magento/Checkout/view/frontend/web/js/model/totals.js

```diff
@@ -14,17 +14,20 @@ define([
     'use strict';
 
     var quoteItems = ko.observable(quote.totals().items),
-        cartData = customerData.get('cart'),
-        quoteSubtotal = parseFloat(quote.totals().subtotal),
-        subtotalAmount = parseFloat(cartData().subtotalAmount);
+        cartData = customerData.get('cart');
 
     quote.totals.subscribe(function (newValue) {
         quoteItems(newValue.items);
     });
 
-    if (quoteSubtotal !== subtotalAmount) {
-        customerData.reload(['cart'], false);
-    }
+    cartData.subscribe(function () {
+        var quoteSubtotal = parseFloat(quote.totals().subtotal),
+            subtotalAmount = parseFloat(cartData().subtotalAmount);
+
+        if (quoteSubtotal !== subtotalAmount) {
+            customerData.reload(['cart'], false);
```

Comment:
it seems that it will starts to poll remote call after every second in those setups where are defined also taxes as `quote.totals().subtotal` returns amount without tax but `cartData().subtotalAmount` is with tax.

Replacing `subtotal` with `subtotal_incl_tax` seems to make them equal

```diff
- var quoteSubtotal = parseFloat(quote.totals().subtotal),
+ var quoteSubtotal = parseFloat(quote.totals().subtotal_incl_tax),

```